extensions/amp-next-page/1.0/test/test-amp-next-page.js

```diff
@@ -432,6 +435,55 @@ describes.realWin(
       });
     });
 
+    describe('initial behavior', () => {
+      let element;
+      let service;
+
+      beforeEach(async () => {
+        element = await getAmpNextPage(
+          {
+            inlineConfig: VALID_CONFIG,
+            scrollTop: 0,
+          },
+          /* no awaiting */ false
+        );
+
+        service = Services.nextPageServiceForDoc(doc);
+        env.sandbox.stub(service, 'getViewportsAway_').returns(2);
+      });
+
+      afterEach(async () => {
+        element.parentNode.removeChild(element);
+      });
+
+      it('awaits first scroll', async () => {
+        element.build();
+        await Promise.resolve();
+        expect(service.pages_.length).to.equal(1);
+        win.dispatchEvent(new Event('scroll'));
+        await Promise.resolve();
+        expect(service.pages_.length).to.equal(3);
+
+        /*await fetchDocuments(service, MOCK_NEXT_PAGE_WITH_RECOMMENDATIONS);
```

Comment:
Hmm what's with this?

Reply:
Had forgotten to hit save removing the comment.